src/components/ChildBountyCreation/CreateChildBounty.tsx

```
@@ -0,0 +1,379 @@
+					</section>
+					<section className='mt-4'>
+						<label className='mb-0.5'>
+							Parent Bounty Id <span className='text-nay_red'>*</span>
+						</label>
+						<Form.Item
+							name='parentBountyIndex'
+							rules={[
+								{
+									message: 'Invalid parent Bounty index',
+									validator(rule, value, callback) {
+										if (callback && isNaN(value)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)